release_build_files/readme.md

```diff
@@ -576,6 +576,9 @@ code.
     -   Messaging (Android): Fixes an issue to receive token when
         initialize the app. 
         ([#667](https://github.com/firebase/firebase-cpp-sdk/pull/667)).
+    -   Auth (Desktop): Fix a crash that would occur if parsing the JSON
+        response from the server failed
```

Comment:
nit: period.

Reply:
The period occurs after the PR link on line 581.